### DISCUSSION TITLE
x64 build fails, wrong App_SharedMemoryPhysics_VR linker property

### DISCUSSION THREAD

**Discussion by jullej** [bulletphysics/bullet3#3532] [General]
Both the debug and release x64 build fails for project App_SharedMemoryPhysics_VR because of wrong "Linker/General/Additional Library Directories" -value. 

The property is set to
....\examples\ThirdPartyLibs\openvr\lib\win32

but should be 
....\examples\ThirdPartyLibs\openvr\lib\win64

Bullet 2.85 master, Windows 10, Visual Studio 2015

**Comment by erwincoumans** [comment]
Yes, only win32 is tested at the moment. It woukd be good (and easy) to support win64 in the future, just by modifying the premake4.lua in Bullet/examples/SharedMemory
